src/app/search/search-filters/loadouts.tsx

```diff
@@ -5,9 +5,34 @@ import { FilterDefinition } from '../filter-types';
 const loadoutFilters: FilterDefinition[] = [
   {
     keywords: 'inloadout',
+
+    // excluding a "format" property causes autogeneration of the simple "is" and "not" stems
+    suggestionsGenerator: ({ loadouts }) =>
+      loadouts
+        // we can't properly quote loadout names if they contain both ' and ", so..
+        // we filter them out. small caveat there for the future "WHY DOESNT THIS WORK" user
+        ?.filter((l) => !(l.name.includes(`'`) && l.name.includes(`"`)))
+        .map((l) => {
+          const loadoutName = l.name.toLowerCase();
+          return loadoutName.includes(`"`)
+            ? `inloadout:'${loadoutName}'`
+            : `inloadout:"${loadoutName}"`;
+        }),
+
     description: tl('Filter.InLoadout'),
-    filter: ({ loadouts }) => {
-      const loadoutItemIds = collectItemsInLoadouts(loadouts);
+    filter: ({ filterValue, loadouts }) => {
+      // the default search:
+      // is:inloadout
+      let selectedLoadouts = loadouts;
+
+      // a search like
+      // inloadout:"loadout name here"
+      if (filterValue !== 'inloadout') {
+        const foundLoadout = loadouts.find((l) => l.name === filterValue);
```

Comment:
Would you want to support a substring of the loadout name?

Reply:
not sure i understand. if you mean while typing,
`cry` 
should recommend 
`inloadout:"deep stone crypt"`
per usual partial matching

Reply:
maybe you mean
`inloadout:cry`
which would be delightful but didn't quite present as low hanging fruit

Reply:
Yeah I meant that latter one - `inloadout:crypt` in the same vein as `perkname:clip`

Reply:
seems doable but will end up a bit hacky with our current low tech suggestion list. be happy to look at implementing it separately

Reply:
Yeah not blocking this at all. No need to generate suggestions for it either, just supporting it in the search.